src/functions.php

```diff
@@ -104,8 +104,12 @@ function debug_resource($value = null)
 function choose_handler()
 {
     $handler = null;
-    if (extension_loaded('curl')) {
+    if (function_exists('curl_multi_exec') && function_exists('curl_exec')) {
```

Comment:
This should have gotten some documentation IMHO, but well, if this is fine for you @mtdowling